plan/explain.go

```diff
@@ -304,3 +304,20 @@ func (p *PhysicalMergeJoin) ExplainInfo() string {
 	}
 	return buffer.String()
 }
+
+// ExplainInfo implements PhysicalPlan interface.
+func (p *TopN) ExplainInfo() string {
+	buffer := bytes.NewBufferString("")
+	for i, item := range p.ByItems {
+		order := "asc"
```

Comment:
We can define explain info for by item. There exists same logic in `sort` and `topn`